quantum-adapters/kiwoom-adapter/src/kiwoom_api/external/naver_news_client.py

```
@@ -0,0 +1,296 @@
+                    return {"items": [], "total": 0, "start": start, "display": display}
+                elif response.status_code == 400:
+                    print(f"❌ 네이버 API 요청 오류 (400): {response.text}")
+                    return {"items": [], "total": 0, "start": start, "display": display}
+                else:
+                    print(f"❌ 네이버 API 오류 ({response.status_code}): {response.text}")
+                    return {"items": [], "total": 0, "start": start, "display": display}
+                    
+        except httpx.TimeoutException:
+            print(f"❌ 네이버 API 타임아웃: {query}")
+            return {"items": [], "total": 0, "start": start, "display": display}
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)